src/store/index.ts

```diff
@@ -25,4 +25,5 @@ export type DesktopSettings = {
   detectedGpu?: GpuType;
   /** The pytorch device that the user selected during installation. */
   selectedDevice?: TorchDeviceType;
+  'Comfy-Desktop.RestoredCustomNodes': boolean;
```

Comment:
We should probably remove this custom node restoration as that was caused by Manager installing custom nodes in wrong location before an eariler version. Now existing users and new users shouldn't have a need for this migration.